OpenRA.Game/Network/Order.cs

```diff
@@ -240,7 +240,7 @@ public byte[] Serialize()
 		{
 			if (IsImmediate)
 			{
-				var ret = new MemoryStream();
+				var ret = new MemoryStream(1 + OrderString.Length + TargetString.Length);
```

Comment:
`w.Write(string)` includes a length prefix so this is at least two too few bytes, but possibly more if the strings are long!

Reply:
Being wrong isn't the end of the world, the stream can still resize. This just gets us in the right ballpark. Alternatively, I could remove it if it's misleading.

Reply:
How about we change this to `3 + OrderString.Length + TargetString.Length`?  That will be correct so long as both strings are less than 255 characters, which should be the most common case.

Reply:
Done. Spelt it `1 + OrderString.Length 1 + TargetString.Length + 1` so it ties up with the things we're writing out like the other call-sites.